src/finput.js

```diff
@@ -197,6 +202,11 @@ class Finput {
     this.setValue(newValue, false);
   }
 
+  isModifierKeyPressed(originalEvent) {
+    const isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
+    return isMac ? originalEvent.metaKey : originalEvent.ctrlKey;
```

Comment:
I've went ahead and added my proposed changes as I think it makes things a bit clearer. i.e. there's only one thing referred to as `event` or `e` which is the original browser event and function docs/comments makes ctrl/meta/command differences clearer.